DOCUMENTATION.md

```diff
@@ -20,7 +20,7 @@ You can install the GOV.UK Notify client package using either the command line o
   <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
   <strong class="govuk-warning-text__text">
     <span class="govuk-warning-text__assistive">Warning</span>
-    The latest version (2.5.0) does not work properly unless your code is asynchronous. Until this problem is fixed you should install the previous version (2.4.0).
+    Version (2.5.0) does not work properly unless your code is asynchronous. This problem has been fixed in version 2.5.2, [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you encounter any issues.
```

Comment:
Suggested changes:

`Version 2.5.0 will only work if your code is asynchronous. We have fixed this issue in version 2.5.2. [Contact us](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you have any problems.`

If it doesn't make sense to say `Contact us` in this context, we can revert to `Contact the GOV.UK Notify team`